learningequality/studio
# Flatten routes 2 - channel list - channel edit modal and channel details modal

## Description

Another PR to flatten routes + related refactoring and cleanup.
I added another commit to shorten route paths (seems that there's no need to have `channels` two times in URL)

Collections will follow soon.

## Steps to Test

Navigate to channel information and channel edit modals from channel lists and public catalog (these modals are also rendered when we navigate to them from `channelEdit` app though I haven't updated its routes in this regard yet, will follow in upcoming PRs). Check that back navigation works properly. Try to reload pages.
